### DISCUSSION TITLE
How to change theme? I want light theme.

### DISCUSSION THREAD

**Discussion by BrianAndersen78** [directus/directus#11692] [Archive]
Am I completely blind, or are there no way to change between light and dark theme? 

I would expect to find a option under my user preferences, but it seem there is none?

**Comment by benhaynes** [comment]
On the User Page under User Preferences... (Top-Right):

![Screen Shot 2022-02-17 at 1 04 35 PM](https://user-images.githubusercontent.com/522079/154543322-24ecf4f9-9607-4a1a-9611-9e7900c3a190.png)

**Reply by ChappIO** [reply]
Sorry for resurrecting this but is there a way to set this globally for all users?

If not, then I'll just make a hook to set it I suppose.

**Reply by paescuj** [reply]
> Sorry for resurrecting this but is there a way to set this globally for all users?
> 
> If not, then I'll just make a hook to set it I suppose.

I don't think so. Yeah, you can pass option when creating new users 👍

**Reply by ChappIO** [reply]
Alright, that's fine. Should be easy enough.

**Comment by BrianAndersen78** [comment]
Thanks 1000 times... I have spend 2 hours looking for that option :) Can't believe how i missed that...
